File: [CODE]/rightoversJAVA/src/main/java/sep3/g3/rightoversjava/model/dto/UserUpdateFirstNameDto.java
package sep3.g3.rightoversjava.model.dto;

public class UserUpdateFirstNameDto {
    private String username;
    private String newFirstName;


    public UserUpdateFirstNameDto(String username, String newFirstName) {
        this.username = username;
        this.newFirstName = newFirstName;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getNewFirstName() {
        return newFirstName;
    }

    public void setNewFirstName(String newFirstName) {
        this.newFirstName = newFirstName;
    }
}

File: [CODE]/rightoversBlazorNWEB/Domain/DTOs/UserUpdateFirstNameDto.cs
namespace Domain.DTOs;

public class UserUpdateFirstNameDto
{
    public string Username { get; }
    public string NewFirstName { get; private set; }

    public UserUpdateFirstNameDto(string username, string newFirstName)
    {
        Username = username;
        NewFirstName = newFirstName;
    }
}